docs/data/sql_functions.yml

```
@@ -431,10 +435,10 @@ string:
   - sql: TRANSLATE(expr, fromStr, toStr)
     table: expr.translate(fromStr, toStr)
     description: |
-      Translate an expr where all characters in fromStr have been replaced with those in toStr.
-
-      If toStr has a shorter length than fromStr, unmatched characters are removed.
+      Translate an expr where all characters in fromStr have been replaced with those in toStr. If toStr has a shorter length than fromStr, unmatched characters are removed.
 
+      E.g., SELECT TRANSLATE3('www.apache.org', 'wapcheorg', ' APCHEcom') or 'www.apache.org'.translate('wapcheorg', ' APCHEcom') returns "   .APACHE.com".
+      
```

```suggestion
      E.g., SELECT TRANSLATE3('www.apache.org', 'wapcheorg', ' APCHEcom') or 'www.apache.org'.translate('wapcheorg', ' APCHEcom') returns '   .APACHE.com'.
      
```
in SQL string usually should be quoted with single quotes